src/T2DMap.cpp

```diff
@@ -2001,6 +2029,7 @@ void T2DMap::paintAreaExits(QPainter& painter, QPen& pen, QList<int>& exitList,
                     painter.drawPolyline(polyLinePoints.data(), polyLinePoints.size());
 
                     if (!getUserDataBool(room->userData, ROOM_UI_DONTSHOWAREAEXITNAME, false)) {
+                        QPointF areaExitNameOffset = parseOffsetString(room->userData.value(ROOM_UI_AREAEXITNAMEOFFSET + "." + itk.key()));
```

Comment:
These things should be done with a `QStringLiteral(`...`)` wrapper so that you are not working on raw string literals - and are explicitly marking them as internal non UI texts that are NOT to be put through the translation system that UI should be...

In this case you should (*I* think) use:
```cpp
    QPointF areaExitNameOffset = parseOffsetString(room->userData.value(QStringLiteral(ROOM_UI_AREAEXITNAMEOFFSET ".%1").arg(itk.key())));
```